script/XUltraLRTMainnet.s.sol

```diff
@@ -145,4 +145,46 @@ contract XUltraLRTMainnet is Script {
         console2.log("addr %s", addr);
         console2.logBytes32(bytes32(uint256(uint160(addr))));
     }
+
+    function transferRemote() public {
+        address deployer = _start();
+
+        XUltraLRT xLRT = XUltraLRT(payable(0x6f987a9495e4C75d27199490bdc12EfA48B0c7F3));
+
+        uint32 destination = 81457;
```

Comment:
can you document that this is only for blast and we will update later?

Reply:
okay, it just a script.